src/pages/Signup/Signup.scss

```
@@ -0,0 +1,149 @@
+* {
+  box-sizing: border-box;
+}
+
+.contentWrapper {
+  margin-top: 3.125rem;
+  margin-left: 0 auto;
+  margin-right: 0 auto;
+  header {
+    font-size: 1.875rem;
+  }
+}
+
+.signUpWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.idContent {
+  display: flex;
+  flex-direction: column;
+  margin-top: 4.375rem;
+  font-size: 0.75rem;
+  .userId {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.nameContent {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+  .isUserName {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.passwordContent {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+  .userPassword {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.passwordCheckContent {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+  .userPasswordCheck {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.phoneNumber {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+  .userPhoneNumber {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.birth {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+  .userBirth {
+    width: 25rem;
+    height: 2.5rem;
+    margin-top: 0.625rem;
+    margin-bottom: 1.875rem;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.footer {
+  position: relative;
+  font-size: 0.875rem;
+  right: 1.25rem;
+  margin-top: 1.25rem;
+
+  .terms {
+    margin-bottom: 0.625rem;
+  }
+  .agreeTerms {
+    margin-bottom: 0.625rem;
+  }
+  .agreeTerms2 {
+    margin-bottom: 0.625rem;
+  }
+  .agreeTerms3 {
+    margin-bottom: 0.625rem;
+  }
```

```suggestion
  .terms {
    margin-bottom: 0.625rem;
  }
```
동일한 스타일을 부여할 것이라면, 각자 다른 className을 줄 필요 없이 하나의 className으로 처리하면 될 것 같네요!!

className의 궁극적인 목적은 스타일을 주기 위함입니다, 항목들이 다르다고 똑같은 스타일을 부여할 것인데 다른 className을 부여할 필요는 없습니다!

---

다른 부분도 마찬가지로 적용되는 코멘트니까 전반적으로 확인 및 수정해주세요~!